node/buildutils.js

```diff
@@ -130,8 +130,16 @@ var downloadArchiveAsync = async function (url, fileName) {
 
     // extract
     mkdir('-p', targetPath);
-    var zip = new admZip(archivePath);
-    zip.extractAllTo(targetPath);
+
+    if (targetPath.endsWith('.zip')) {
+        var zip = new admZip(archivePath);
+        zip.extractAllTo(targetPath);
+    }
+    else if (targetPath.endsWith('.tar.gz')) {
+        run(`tar --extract --gzip --file="${archivePath}" --directory="${targetPath}"`);
```

Comment:
Should we check that 'tar' exist on the OS?

Reply:
I believe if it's not, it will fail with verbose message 😃

Reply:
AFAIK, tar comes preinstalled on all distros that we currently support. Same goes for MacOS.